.github/workflows/codecov.yaml

```diff
@@ -182,3 +182,29 @@ jobs:
           fail_ci_if_error: false
           verbose: true
           files: merged.xml
+
+      - name: Get current date
+        id: run_date
+        run: |
+            echo "run_date=$(date --iso-8601=minutes)" >> "$GITHUB_OUTPUT"
+            echo "run_date_short=$(date +'%Y-%m-%d')" >> "$GITHUB_OUTPUT"
+            echo "run_date_year=$(date +'%Y')" >> "$GITHUB_OUTPUT"
+            echo "run_date_month=$(date +'%m')" >> "$GITHUB_OUTPUT"
+
+      - name: Generate Coverage Report
+        if: always()
+        run: |
+          pip install xlsxwriter ijson
+          ./scripts/twister -E testplan.json
```

Comment:
This will not work, please look at the workflow link I provided. running twister here requires a different runner, you do not have the dependencies needed for twister in the default runner. see https://github.com/zephyrproject-rtos/zephyr-testing/actions/runs/7891137291/workflow

Reply:
You're right. I needed to take a look at this part of workflow again.